### ISSUE TITLE
[libc++] no member named 'ends_with' in namespace 'std::ranges'

### ISSUE THREAD

**Issue by shafik** [llvm/llvm-project#67708]
Looks like the test: `libcxx/benchmarks/algorithms/ranges_ends_with.bench.cpp` is failing on the build bots: https://buildkite.com/llvm-project/libcxx-ci/builds/30085#018adc40-6c02-4cd5-837a-2af2dfe0cd25

This is the error:

```console
libcxx/benchmarks/CMakeFiles/ranges_ends_with_libcxx.dir/algorithms/ranges_ends_with.bench.cpp.o -c /home/libcxx-builder/.buildkite-agent/builds/google-libcxx-builder-b863daf6ebb6-1/llvm-project/libcxx-ci/libcxx/benchmarks/algorithms/ranges_ends_with.bench.cpp
--
  | /home/libcxx-builder/.buildkite-agent/builds/google-libcxx-builder-b863daf6ebb6-1/llvm-project/libcxx-ci/libcxx/benchmarks/algorithms/ranges_ends_with.bench.cpp:29:43: error: no member named 'ends_with' in namespace 'std::ranges'
  | 29 \|     benchmark::DoNotOptimize(std::ranges::ends_with(begin1, end1, begin2, end2));
  | \|                              ~~~~~~~~~~~~~^
  | /home/libcxx-builder/.buildkite-agent/builds/google-libcxx-builder-b863daf6ebb6-1/llvm-project/libcxx-ci/libcxx/benchmarks/algorithms/ranges_ends_with.bench.cpp:47:43: error: no member named 'ends_with' in namespace 'std::ranges'
  | 47 \|     benchmark::DoNotOptimize(std::ranges::ends_with(begin1, end1, begin2, end2));
  | \|                              ~~~~~~~~~~~~~^
  | /home/libcxx-builder/.buildkite-agent/builds/google-libcxx-builder-b863daf6ebb6-1/llvm-project/libcxx-ci/libcxx/benchmarks/algorithms/ranges_ends_with.bench.cpp:65:43: error: no member named 'ends_with' in namespace 'std::ranges'
  | 65 \|     benchmark::DoNotOptimize(std::ranges::ends_with(begin1, end1, begin2, end2));
  | \|                              ~~~~~~~~~~~~~^
  | /home/libcxx-builder/.buildkite-agent/builds/google-libcxx-builder-b863daf6ebb6-1/llvm-project/libcxx-ci/libcxx/benchmarks/algorithms/ranges_ends_with.bench.cpp:83:43: error: no member named 'ends_with' in namespace 'std::ranges'
  | 83 \|     benchmark::DoNotOptimize(std::ranges::ends_with(begin1, end1, begin2, end2));
  | \|                              ~~~~~~~~~~~~~^
  | /home/libcxx-builder/.buildkite-agent/builds/google-libcxx-builder-b863daf6ebb6-1/llvm-project/libcxx-ci/libcxx/benchmarks/algorithms/ranges_ends_with.bench.cpp:102:43: error: no member named 'ends_with' in namespace 'std::ranges'
  | 102 \|     benchmark::DoNotOptimize(std::ranges::ends_with(begin1, end1, begin2, end2));
  | \|                              ~~~~~~~~~~~~~^
  | 5 errors generated.


```

**Comment by ldionne** [comment]
Yeah, thanks for the heads up. This has been happening since 10edd5d9436153ace82009a04900ac67d3adc202.

**Comment by ldionne** [comment]
Fixed by dd788af74a33b0014b90e8fccf3fe377ee67cb29.
